package.json

```diff
@@ -26,7 +26,7 @@
     "nektos/act"
   ],
   "scripts": {
-    "postinstall": "node scripts/postinstall.js 0.2.48",
+    "postinstall": "node scripts/postinstall.js 0.2.61",
```

Comment:
Yes since the postinstall script automatically runs after the `npm install... `, it means that for users who don't manually/separately install act and use the `ACT_BINARY`env variable to point to their version, they will be automatically using the version specified in the postinstall script

Reply:
Correct. I had added it to fix https://github.com/kiegroup/act-js/issues/23

Reply:
Ah, I see. We haven't faced this because we have always explicitly used the ACT_BINARY so that we control the exact version.